stubs/beautifulsoup4/METADATA.toml

```
@@ -1,4 +1,5 @@
 version = "4.12.*"
+upstream_repository = "https://git.launchpad.net/beautifulsoup"
```

Maybe this would be more useful? I generally don't care too much about the git log when I'm looking at an upstream repository to review a typeshed PR, I just wanna see the source 😆

```suggestion
upstream_repository = "https://git.launchpad.net/beautifulsoup/tree"
```

---

Agreed. For example docutils' sourceforge link goes directly to the code tab.
I'll update the launchpad link in pytz comment too